index.html

```diff
@@ -91,16 +91,16 @@ <h2 class="card-header">Forum</h2>
           <i class="fa-solid fa-chalkboard-user main-icon"></i>
           <h2 class="card-header">Workshop</h2>
           <p class="card-p">
-            Try creating your own work using open source license rather than just
-            watching.
+            Try creating your own work using open source license rather than
+            just watching.
           </p>
         </div>
         <div class="card">
```

Comment:
I kindly suggest you use the semantic ARTICLE tag instead of DIV, this is the ideal tag for repetitive content.

Reply:
Done. Thank you for your reviews